lib/internal/bootstrap/web/exposed-window-or-worker.js

```diff
@@ -55,6 +55,9 @@ exposeLazyInterfaces(globalThis, 'perf_hooks', [
 
 defineReplaceableLazyAttribute(globalThis, 'perf_hooks', ['performance']);
 
+// https://html.spec.whatwg.org/multipage/system-state.html#the-navigator-object
+exposeLazyInterfaces(globalThis, 'internal/navigator', ['Navigator', 'navigator']);
```

Comment:
`Object.getOwnPropertyDescriptor(this, 'navigator').enumerable` is `true` in browsers
```suggestion
exposeLazyInterfaces(globalThis, 'internal/navigator', ['Navigator']);
exposeLazyProperties(globalThis, 'internal/navigator', ['navigator']);
```

Reply:
I guess we would need a `defineNonReplaceableLazyAttribute` util

Reply:
We don't have a `exposeLazyProperties` attribute. This is a part of Node.js, I'm not familiar with, any suggestions?

Reply:
Why not use `defineLazyProperties`?

Reply:
It seems `defineReplaceableLazyAttribute` takes a 4th argument `writable` that solves our problem

```js
exposeLazyInterfaces(globalThis, 'internal/navigator', ['Navigator']);
defineReplaceableLazyAttribute(globalThis, 'internal/navigator', ['navigator'], false);
```

```
➜  node git:(navigator-platform) ✗ out/Release/node
Welcome to Node.js v21.0.0-pre.
Type ".help" for more information.
> Object.getOwnPropertyDescriptor(global, 'navigator')
{
  get: [Function: get navigator],
  set: undefined,
  enumerable: true,
  configurable: true
}
```

Reply:
Good catch!